storage/Filesystems/FilesystemImpl.cc

```diff
@@ -783,16 +791,64 @@ namespace storage
 
 
     Text
+    Filesystem::Impl::do_rename_etc_fstab_text(const Device* lhs, const string& mountpoint,
```

Comment:
Again: Rename file vs. rename **in** file. This should be `do_rename_in_etc_fstab_text()`.